### DISCUSSION TITLE
How to save last column sorted on AutoType?

### DISCUSSION THREAD

**Discussion by BanCrash** [keepassxreboot/keepassxc#8162] [Q&A]
Hi, I think this will be a feature request but first I want to ask just to make sure that it isn't something already present. When I sort by Title on AutoType, after closing it and opening it again it will be sorted again by Group instead remembering that on my last use I sorted it by Title. I search on issues, discussions and on the settings but I can't find out a way to remember the last sorted column each time you open AutoType.

Regards and thanks.

**Comment by droidmonkey** [comment]
It's a feature request, not implemented yet

**Reply by BanCrash** [reply]
Okay then, thanks!
